libs/src/api/serviceProvider.js

```diff
@@ -82,35 +93,46 @@ class ServiceProvider extends Base {
       creatorNodes = creatorNodes.filter(node => !blacklist.has(node.endpoint))
     }
 
-    // Filter to healthy nodes
-    creatorNodes = (await Promise.all(
+    // Get sync status for all nodes and filter out unhealthy (unresponsive) nodes.
+    const healthyCreatorNodesSyncStatus = {}
+    await Promise.all(
       creatorNodes.map(async node => {
         try {
-          const { isBehind, isConfigured } = await this.creatorNode.getSyncStatus(node.endpoint)
-          return isConfigured && isBehind ? false : node.endpoint
+          const syncStatus = await this.creatorNode.getSyncStatus(node.endpoint)
+          healthyCreatorNodesSyncStatus[node.endpoint] = syncStatus
         } catch (e) {
-          return false
+          // Ignore unhealthy nodes.
+          console.error(`get sync status failed ${node.endpoint}`)
```

Comment:
maybe this should we warning/debug? it's not really an *error* since our system is supposed to tolerate this?